runelite-client/src/main/java/net/runelite/client/plugins/instancemap/InstanceMapOverlay.java

```diff
@@ -210,6 +181,17 @@ public Dimension render(Graphics2D graphics)
 		return new Dimension(image.getWidth(), image.getHeight());
 	}
 
+	/**
+	 * Get the files for the current viewed plane
+	 *
+	 * @return
```

Comment:
There should be some documentation for return otherwise it will just throw JavaDoc warnings what is even worse than not having documentation here at all